dm/checker/checker.go

```diff
@@ -250,6 +253,20 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 		}
 	}
 
+	if _, ok := c.checkingItems[config.ConnAmountChecking]; ok {
+		if len(c.stCfgs) > 0 && len(c.instances) > 0 {
+			switch c.stCfgs[0].Mode {
+			case config.ModeAll:
+				c.checkList = append(c.checkList, checker.NewLoaderConnAmountCheker(c.instances[0].targetDB, c.stCfgs))
```

Comment:
as this check is somthing about connection count,  do we need to put this item in the head of the check queue?

Reply:
Will fix it.

Reply:
Checker list is parallel. So it doesn't matter.

https://github.com/pingcap/tiflow/blob/536e18274b90934d7611caca25e40f015e1c163a/dm/pkg/checker/real_checker.go#L131-L139